### DISCUSSION TITLE
Something went wrong! file an issue at https://tiny.one/readme-stats

### DISCUSSION THREAD

**Discussion by janithsuraweera** [anuraghazra/github-readme-stats#4040] [General]
Something went wrong! file an issue at https://tiny.one/readme-stats
Could not resolve to a User with the login of 'janithsuraweera'.
![Screenshot 2024-11-21 132246](https://github.com/user-attachments/assets/921bbacb-f93e-47d0-baaa-93c49e03962e)

I use the same username to create Md file but and not display my[readme.md] file in my profile front.

**Comment by qwerty541** [comment]
According to your screenshot, the problem is that you are using "janith_suraweera" instead of your actual login "janithsuraweera".

I see that you already fixed a card in your profile, so I'm considering that the issue is resolved.
